templates/taxi/driver_form.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+
+{% block content %}
+  <h1>{{ object|yesno:"Update license number of,Create new driver" }} {{ driver.username }}</h1>
```

Comment:
The `yesno` filter is typically used with boolean values. Ensure that `object` is a boolean indicating whether the form is for updating (True) or creating (False) a driver. If `object` is not a boolean, consider using a different approach to determine the form's purpose.